### ISSUE TITLE
Nunca, nunca use um banco de dados em um container :)

### ISSUE THREAD

**Issue by ezfranca** [rodrigo-tsuru/db2-docker#1]
;)

**Comment by rodrigo-tsuru** [comment]
Conte-me mais. Neste primeiro momento não concordo com você. Se parar pra pensar o conceito do Docker é antigo (vide Zones do Solaris e WPAR no AIX). Todos os bancos suportavam e rodavam normalmente nesses ambientes.

**Comment by ezfranca** [comment]
lol

Conceito é uma coisa. Na prática, todo mundo q usou se fodeu. E não falo do conceito em si. Tô falando do ecossistema Docker. 

Todos, todos meus amigos, pelo menos alguma vez se fodeu usando db no docker. Em uma arquitetura de micro servićos, o que temos usado é no máximo o banco read only como cache ou subset das consultas das API's, quando aplicável.

[https://myopsblog.files.wordpress.com/2017/01/archer.jpg](url)

**Comment by rodrigo-tsuru** [comment]
Sim, toda tecnologia tem seus prós e contras. Ele cabe como uma luva pra microserviços, mas não atende tudo pra banco de dados SQL (por enquanto). Mas não dá pra chegar a concluir: "não utilize banco de dados em container". Pensa comigo: o DB2 é fácil de instalar? E de configurar? E de subir? O melhor jeito de abstrair isso tudo pro desenvolvedor é o docker (por enquanto).
